internal/code-generator/generator/generator_test.go

```diff
@@ -244,8 +241,29 @@ func (s *GeneratorTestSuite) TestGenerationFiles() {
 			wantOut:     mimeTypeGold,
 		},
 	}
+}
 
-	for _, test := range tests {
+func (s *GeneratorTestSuite) TearDownSuite() {
+	if s.cloned {
+		err := os.RemoveAll(s.tmpLinguist)
+		assert.NoError(s.T(), err)
+	}
+}
+
+// UpdateGeneratorTestSuiteGold is a Gold results generation automation.
+// It should only be enabled&run manually on every new linuguist verision
```

Comment:
```suggestion
// It should only be enabled&run manually on every new Linguist version
```

Reply:
Rather than commenting this out, can we put in a flag?

```go
var updateGold = flag.Bool("update_gold", false, "Update golden test files")
…
func (s *GeneratorTestSuite) TestUpdateGeneratorTestSuiteGold() {
   if !*updateGold {
      s.T().Skip()
   }
   …
}
```

or words to that effect?

Reply:
Having runtime switch instead of compile-time sounds really good for convenience, thank your for suggestion!
But at any pace, just having this automation is already very useful for maintainer :)